src/encoded/static/scss/encoded/modules/_forms.scss

```diff
@@ -90,10 +96,12 @@ input[type="text"], input[type="password"], input[type="date"], input[type="date
 
 .search-wrapper {
 	position: 0;
+	background-color: white;
+	border-radius: 20px;
+	padding: 2px !important;
```

Comment:
I'm not sure that you need the `!important` here.

Reply:
Good point. Corrected. I think I needed it at one point but that is no longer the case.